config/prod.exs

```diff
@@ -3,4 +3,5 @@ use Mix.Config
 config :glass_factory_api, GlassFactoryApi.ApiClient,
   subdomain: System.fetch_env!("GLASSFACTORY_SUBDOMAIN"),
   user_token: System.fetch_env!("GLASSFACTORY_USER_TOKEN"),
-  user_email: System.fetch_env!("GLASSFACTORY_USER_EMAIL")
+  user_email: System.fetch_env!("GLASSFACTORY_USER_EMAIL"),
+  api_domain: System.fetch_env!("GLASSFACTORY_API_DOMAIN")
```

Comment:
🎨
Maybe it's not in the scope of this PR but WDYT about using one key for the whole "domain" concept?
For example, `api_domain` could already include the `subdomain` in itself

Reply:
I have two separated variables because, in some environment, we need to use `HTTP` and others `HTTPS` protocols and I need the subdomain in the request headers as well 😞 .

As you suggested, I'm changing `GLASSFACTORY_API_DOMAIN` to `GLASSFACTORY_API_URL`.

Reply:
LGTM 👍